### DISCUSSION TITLE
What's the [1] behind the basename on the title bar?

### DISCUSSION THREAD

**Discussion by TheCGDF** [d2phap/ImageGlass#1297] [Q&A] [answered]
When open picture on my Android phone, there will be a `[1]` behind the basename:
![ImageGlass_HvvJBYWErI](https://user-images.githubusercontent.com/17256673/155937907-5e9897cc-2fe4-455b-99d4-e3c1716694b0.jpg)
Sometimes it will be `[2]`.

The raw name does not have `[1]`:
![image](https://user-images.githubusercontent.com/17256673/155937999-55db9326-bd32-4cf7-8fe5-5d5da76614fb.png)

It is not related to the index of pictures, it is still `[1]` when I open second picture.

**Comment by fire-eggs** [comment, marked as answer]
This is Microsoft. Windows Explorer actually makes a copy of the file to your local machine and adds the [1] to it. If you're seeing [2] it's because you have two files with the same name in the place where Windows Explorer copied it.

If you turn off the "Display basename" option in ImageGlass settings, you'll see the full path of where Windows Explorer copied the file. For example [me viewing a file on my Android tablet]:

![image](https://user-images.githubusercontent.com/9809727/157102348-7fc53d66-14bf-4cc8-8eb3-5956b6e743e3.png)

**Reply by TheCGDF** [reply]
Okay... Thank you. It's so weird...

**Reply by fire-eggs** [reply]
Yes, it's very weird.
